core/src/main/java/com/redhat/lightblue/client/request/LightblueRequest.java

```diff
@@ -39,6 +39,7 @@ public HttpMethod getHttpMethod() {
     /**
      * Requst body as string, defaults to getBodyJson().toString()
      */
+    @Deprecated
```

Comment:
getBodyJson should be used instead. This uses a default global object mapper so it can't be configured. I'll add a comment.